Tests/test_imaging_leaks.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+
+from helper import unittest, PillowTestCase
+import sys
+from PIL import Image, ImageFilter
+
+min_iterations = 100
+max_iterations = 1000
+
+@unittest.skipIf(sys.platform.startswith('win32'), "requires Unix or MacOS")
+class TestWebPLeaks(PillowTestCase):
+
+    def _get_mem_usage(self):
+        from resource import getpagesize, getrusage, RUSAGE_SELF
+        mem = getrusage(RUSAGE_SELF).ru_maxrss
+        return mem * getpagesize() / 1024 / 1024
+
+    def _test_leak(self, min_iterations, max_iterations, fn, *args, **kwargs):
+        start_mem = self._get_mem_usage()
```

Comment:
There is no need in `start_mem` anymore. `mem < mem_limit` constraint is enough.